### PR TITLE
Metrics api endpoint

### PR DESCRIPTION
**Pull request by lbliii**
this includes the endpoints without use cases so you can see what we've got so far.

### CODE REVIEW

File: content/api/api-metrics.md [hunk 1]
```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
...
+| startTime | query | date-time | Start time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| endTime | query | date-time | End time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| rollup | query | Array[string] | Determines frequency of metric sampling. Options are: ZERO, PT5M, PT1H, PT24H. |
+| showValues | query | boolean | Provides the values for the metric statistics (if set to true) or only displays zeroes for the statistics (if set to false). |
+
+
+### Request URL
+
+```
+https://app.metricly.com/metrics/statistics?fqn=aws.applicationelb.httpcode_elb_4xx_count&startTime=2020-01-30T01%3A00%3A00-05%3A00&endTime=2020-01-31T01%3A00%3A00-05%3A00&rollup=PT5M&showValues=false
```
Comment (rclagett) [inline content/api/api-metrics.md]: Use copy-able code span here too.

URL should be https://app.metricly.com/metrics/statistics?fqn={fqn}&startTime={startTime}&endTime={endTime}&rollup={rollup}&showValues={showValues}

File: content/api/api-metrics.md [hunk 2]
```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
...
+```
+
+### CURL
+
+In the following CURL example
```
Comment (rclagett) [inline content/api/api-metrics.md]: Forgot to explain the example.

File: content/api/api-metrics.md [hunk 3]
```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
...
+
+```
+
+{{% /expand %}}
+
+---
+
+## POST to /metrics/statistics
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/metrics/getMetricStatisticsPageUsingPOST" theme="success" >}} POST {{< /button >}} This endpoint is deprecated.
```
Comment (rclagett) [inline content/api/api-metrics.md]: This endpoint is deprecated but we don't have a better endpoint suggest as a replacement at this time.
```suggestion
{{< button href="https://app.metricly.com/swagger-ui.html#!/metrics/getMetricStatisticsPageUsingPOST" theme="success" >}} POST {{< /button >}} Use this endpoint to get metric statistics for given set of elements. Note this is a deprecated method and could be removed at a later date.  
```

File: content/api/api-metrics.md [hunk 4]
```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
...
+| endTime | query | date-time | End time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| rollup | query | Array[string] | Determines frequency of metric sampling. Options are: ZERO, PT5M, PT1H, PT24H. |
+| showValues | query | boolean | Provides the values for the metric statistics (if set to true) or only displays zeroes for the statistics (if set to false). |
+|elementIds   | body   | Array[string]  | List of elementIds. |
+
+
+### Request URL
+
+```
+https://app.metricly.com/metrics/statistics?fqn=aws.applicationelb.requestcountpertarget&startTime=2020-02-04T00%3A25%3A46.872Z&endTime=2020-02-05T00%3A25%3A46.872Z&rollup=PT5M&showValues=false
```
Comment (rclagett) [inline content/api/api-metrics.md]: Use the copy-able code span here too.

URL should be https://app.metricly.com/metrics/statistics?fqn={fqn}&startTime={startTime}&endTime={endTime}&rollup={rollup}&showValues={showValues}

File: content/api/api-metrics.md [hunk 5]
```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
...
+| elasticsearchQuery | body | json | A JSON query. |
+| aggregation | query | string | Define aggregation method for query. Options are: avg, sum, cnt, min, max, stddev. |
+| statistic | query | string | Define statistic. Options include: val, min, avg, max, cnt, sum, actual. |
+| rollup | query | Array[string] | Determines frequency of metric sampling. Options are: ZERO, PT5M, PT1H, PT24H. |
+
+
+### Request URL
+
+```
+https://app.metricly.com/metrics/crosselementagg?aggregation=avg&statistic=actual&rollup=ZERO
```
[the remark this answers is not in the document]
Comment (rclagett) [reply content/api/api-metrics.md]: Still need the copyable code span.

File: content/api/api-metrics.md [hunk 6]
```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
...
+| Parameter | Parameter Type | Data Type | Description |
+|--------------------|----------------|---------------|------------------------------------------------------------------------------------|
+| elasticsearchQuery | body | json | A JSON query. |
+| term | path | string | The element field you want to aggregate. |
+
+
+### Request URL
+
+```
+https://app.metricly.com/metrics/elasticsearch/metricAgg/fqn
```
[the remark this answers is not in the document]
Comment (rclagett) [reply content/api/api-metrics.md]: Still need the copyable code span.

File: content/api/api-metrics.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (lbliii) [reply content/api/api-metrics.md]: the first time i saw this i was confused -- both code styles have the copy icon, but now i get it haha the condensed yellow one.

[the remark this answers is not in the document]
Comment (rclagett) [reply content/api/api-metrics.md]: Ah, I didn't even see the copy button in the bigger span, sorry about that!

File: content/api/api-metrics.md [hunk 7]
```diff
@@ -7,112 +7,935 @@ author: Lawrence Lane
...
+|elementIds   | body   | Array[string]  | List of elementIds. |
+
+
+### Request URL
+
+`https://app.metricly.com/metrics/statistics?fqn={fqn}&startTime={startTime}&endTime={endTime}&rollup={rollup}&showValues={showValues}`
+
+### CURL
+
+In the following CURL example grabs statistic on the **fqn** `aws.applicationelb.requestcountpertarget` matching two **elementIds** `["861111fd-a348-3f05-92e1-a681d7d3d9e4", "6bdf4fd1-7134-1111-9c36-17eb5f628e46"]`. The **rollup** value is PT5M.
```
Comment (rclagett) [inline content/api/api-metrics.md]:
```suggestion
In the following CURL example, statistics are retrieved for a metric having the **fqn** `aws.applicationelb.requestcountpertarget` for two **elementIds** `["861111fd-a348-3f05-92e1-a681d7d3d9e4", "6bdf4fd1-7134-1111-9c36-17eb5f628e46"]`. The **rollup** value is PT5M.
```

**Pull request merged**